cif_core.dic

```diff
@@ -25109,21 +25109,142 @@ save_atom_type_scat.versus_stol_list
          '_atom_type_scat_versus_stol_list'
          '_atom_type.scat_versus_stol_list'
 
-    _definition.update            2013-04-17
+    _definition.update            2023-04-10
     _description.text
 ;
-    List of scattering factors as a function of sin(θ)/λ.
-    List has the form [[<stol value 1> <scatfac 1>] [<stol value 2> <scatfac
-    2>] ....] in increments of 0.01, increasing from 0.0.
+    A table of scattering factors as a function of sin(θ)/λ (sin theta over
+    lambda, stol). This table should be well-commented to indicate the items
+    present. Regularly formatted lists are strongly recommended.
 ;
     _name.category_id             atom_type_scat
     _name.object_id               versus_stol_list
+    _type.purpose                 Describe
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Text
+
+save_
+
+save_ATOM_TYPE_SCAT_TABLE
+
+    _definition.id                ATOM_TYPE_SCAT_TABLE
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-04-10
+    _description.text
+;
+    The CATEGORY of data items used to list atomic scattering factor values for
+    use in crystallographic structure studies.
+;
+    _name.category_id             ATOM
+    _name.object_id               ATOM_TYPE_SCAT_TABLE
```

Comment:
Since the parent category is now atom, we could probably rename the category to `ATOM_SCAT_TABLE`.  However, I wonder if the name is not too generic. That is, do any other scattering tables come in mind instead of the stol one?

Reply:
I think it is no less generic than `atom_type_scat_table`. I'm comfortable that the new name is OK.

Reply:
I was simply wondering maybe something like `ATOM_SCAT_VERSUS_STOL` might be better, e.g.:

```
loop_
_atom_scat_versus_stol.atom_type
_atom_scat_versus_stol.stol_value
_atom_scat_versus_stol.scat_value
Ac  0.00  89.00000
Ac  0.01  88.91457
Ac  0.02  88.66288
# ...
```

Reply:
@jamesrhester , are you OK with the `ATOM_SCAT_VERSUS_STOL` suggestion?

Reply:
yes, I'm OK with that.

Reply:
Ok, renamed the category and related data names.